benchmarks/fogbench/__main__.py

```diff
@@ -184,6 +208,30 @@ def read_out_file(_file=None, _keep=False):
     print('Result: %s\n' % response.code)
 
 
+def display_statistics(stats_type):
+    print(u"{} statistics: ".format(stats_type))
+    global _START_TIME
+    global _END_TIME
+    global _TOT_MSGS_TRANSFERRED
+    global _TOT_BYTE_TRANSFERRED
+    global _NUM_ITERATED
+    if stats_type == 'total' or stats_type == 'st':
+        print(u"Start Time::{}".format(_START_TIME))
```

Comment:
sorry, we should display start time  with ms system time (without timezone) as it looks with `datetime.now()`